Mathlib/Algebra/Group/Pi/Units.lean

```diff
@@ -0,0 +1,39 @@
+/-
+Copyright (c) 2024 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.Algebra.Group.Pi.Basic
+import Mathlib.Algebra.Group.Units.Defs
+import Mathlib.Algebra.Group.Equiv.Basic
+import Mathlib.Util.Delaborators
+
+/-! # Units in pi types -/
+
+variable {ι} {M : ι → Type*} [∀ i, Monoid (M i)] {x : Π i, M i}
+
+open Units in
+/-- The monoid equivalence between units of a product,
+  and the product of the units of each monoid. -/
```

Comment:
nit: I think indenting wrapped docstrings is going out of fashion, but indeed we have no clear rule on it.

Reply:
Sure, I have no opinion on this; fixed.